src/interaction/interaction_elph.h

```diff
@@ -27,50 +32,52 @@
  * 3) for polar materials, adds the long-range Frohlich interaction.
  */
 // TODO: add flag to let user decide whether to use or not polar corrections
-class InteractionElPhWan {
+class InteractionElPhWan: public InteractionElPhBase {
+
+private:
 
-  Crystal crystal;
-  PhononH0 phononH0;
-  int numPhBands, numElBands, numWsR1Vectors, numWsR2Vectors;
-  std::vector<Eigen::Tensor<double, 3>> cacheCoupling;
-  bool usePolarCorrection = false;
-  const int phaseConvention;
+    ComplexView4D elPhCached;
+    ComplexView5D couplingWannier_k;
 
-  // kokkos objects for GPU accelerated elph coupling
-  ComplexView4D elPhCached;
```

Comment:
it would be great to keep this notation of "wigner-seitz vector" R1 and R2 if possible. Te change in notation was, if I remember correctly, just Re->WS1 and Rp->WS2. 

This change enables me to do also interpolation of JDFTx matrix elements, which are in a basis Re, Re'.